NGINX Plus Release 34 (#343)
nginx/documentation: content/nginx/deployment-guides/single-sign-on/entra-id.md

---

> description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Microsoft Entra ID (formerly Azure Active Directory) as the identity provider (IdP).
> doctypes:
> - task
> title: Single Sign-On with Microsoft Entra ID
> toc: true
> weight: 400

Reviewer: ```suggestion
description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Microsoft Entra ID (formerly Azure Active Directory) as the identity provider (IdP).
type:
- how-to
product: NGINX-PLUS
title: Single Sign-On with Microsoft Entra ID
toc: true
weight: 400
```
